kymatio/scattering2d/backend/jax_backend.py

```diff
@@ -0,0 +1,119 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko, Muawiz Chaudhary
```

Comment:
this author list is wrong. we should probably remove all those lines from the full repo, because they do not give any information beyond the authors and contributors list we already have. so i'd remove this here and then do a separate PR to remove it everywhere else

Reply:
agree